docker/Dockerfile

```diff
@@ -30,10 +31,11 @@ WORKDIR /invidious
 RUN addgroup -g 1000 -S invidious && \
     adduser -u 1000 -S invidious -G invidious
 COPY ./assets/ ./assets/
-COPY ./config/config.yml ./config/config.yml
+COPY --chown=invidious ./config/config.yml ./config/config.yml
```

Comment:
I'd prefer to have this owned by root, any reason invidious needs write permissions to its config file?

Reply:
invidious will write to the config if you change them in the web ui as admin.

Reply:
Is this something we want to support? The config file is stored in a container after all and is reverted once the container gets recreated.

Reply:
btw, where can I find the admin UI? ;)

Reply:
for example I use it to enable registration temporarily when needed. also the invidious errors out with "looks like you found a bug".